core/tools/nginx/nginx.conf

```diff
@@ -55,6 +55,14 @@ http {
             proxy_pass http://127.0.0.1:8000;
         }
 
+        location /beacon {
+            rewrite ^/beacon$ /beacon/ redirect;
+            rewrite ^/beacon/(.*)$ /$1 break;
+            proxy_pass http://127.0.0.1:9111;
+            proxy_set_header X-Real-IP $remote_addr;
+            proxy_set_header X-Interface-Ip $server_addr;
```

Comment:
As a curiosity, what's the difference between the real and the interface IP?

Reply:
Real-Ip is the client IP. Interface IP is the IP that was reached in the backend. (usually 192.168.2.2, or the pi's wifi ip)